plugins/postcss-nesting/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changes to PostCSS Nesting
 
+### Unreleased (patch)
+
+- Edition `2024-02`:
+  - Do not rewrite selectors for declarations in conditional rules to a form with `:is()`
```

Comment:
Is this indent intentional?

Also, I assume you want to patch this after the major?

Reply:
Before, the major.

This is a minimally breaking change.
(Any change theoretically is a breaking change for someone.)

It is only breaking when having something like this :

```css
a, .link {
  @media (min-width: 64rem) {
    color: blue;
  }
}

a {
  color: red;
}
```

Before this change `blue` would have specificity `[0, 1, 0]` because of `:is()` and would overrule `red`.

After this change `blue` will have specificity `[0, 0, 1]` and `red` will have win because it appears later.

So it only affects declarations in nested conditionals that became more specific as a result of `:is()` **and** had lower specificity declarations appearing after it.

User that do depend on the `:is()` behavior would need to opt-in by writing :

```css
a, .link {
  @media (min-width: 64rem) {
    & {
      color: blue;
    }
  }
}

a {
  color: red;
}
```

Reply:
Ahh, makes sense!